libcxx/test/std/numerics/numeric.ops/numeric.ops.gcd/gcd.pass.cpp

```diff
@@ -67,12 +68,14 @@ T basic_gcd(T m, T n) {
 template <typename Input>
 void do_fuzzy_tests() {
   std::mt19937 gen(1938);
-  std::uniform_int_distribution<Input> distrib;
+  using DistIntType         = std::conditional_t<sizeof(Input) == 1, int, Input>; // See N4981 [rand.req.genl]/1.5
+  constexpr Input max_input = std::numeric_limits<Input>::max();
+  std::uniform_int_distribution<DistIntType> distrib(0, max_input);
```

Comment:
We need to keep `Input` as is. Your commit message says that the standard prohibits `int8_t`, but the wording is "undefined", not "ill-formed", meaning that we're allowed to provide `int8_t` as a documented extension, which libc++ does.

Reply:
Sorry @cjdb , I merged before you had time to comment. This wasn't intentional.

Reply:
But this is the `libcxx/test/std` subdirectory, which is portable. If you want to test a libc++ extension, that should be guarded with the libc++ macro (which I would be fine with).

Reply:
Oh, this isn't testing `uniform_int_distribution`? Never mind then.